repos/fdbt-site/src/pages/api/updateFareTriangle.ts

```diff
@@ -0,0 +1,158 @@
+import { NextApiResponse } from 'next';
+import { putDataInS3 } from '../../data/s3';
+import { NextApiRequestWithSession, ErrorInfo, UserFareStages } from '../../interfaces';
+import { redirectToError, redirectTo } from '../../utils/apiUtils';
+import {
+    INPUT_METHOD_ATTRIBUTE,
+    CSV_UPLOAD_ATTRIBUTE,
+    MATCHING_JSON_ATTRIBUTE,
+    PRODUCT_AND_SERVICE_ID_ATTRIBUTE,
+} from '../../constants/attributes';
+import { getFormData, processFileUpload } from '../../utils/apiUtils/fileUpload';
+import { getSessionAttribute, updateSessionAttribute } from '../../utils/sessions';
+import { WithIds, ReturnTicket, SingleTicket } from '../../../shared/matchingJsonTypes';
+import { faresTriangleDataMapper, setCsvUploadAttributeAndRedirect } from './csvUpload';
+
+const errorId = 'csv-upload';
+
+export const config = {
+    api: {
+        bodyParser: false,
+    },
+};
+
+const getNamesOfFareZones = (ticket: WithIds<SingleTicket> | WithIds<ReturnTicket>) => {
```

Comment:
Could we make a unit test for this? Where we pass it a Single Ticket and Return Ticket, and just ensure it gets the right results.

Reply:
Will add unit test.

Reply:
Unit test done :+1: